runtime/lua/vim/lsp/handlers.lua

```diff
@@ -111,6 +111,7 @@ M[ms.client_registerCapability] = function(_, result, ctx)
   local client_id = ctx.client_id
   local client = assert(vim.lsp.get_client_by_id(client_id))
 
+  --- @diagnostic disable-next-line: invisible
   client.dynamic_capabilities:register(result.registrations)
```

Comment:
To fix this we should just remove `@private` from register and replace with `@package`.